src/k8s-extension/azext_k8s_extension/vendored_sdks/models.py

```diff
@@ -6,4 +6,4 @@
 # --------------------------------------------------------------------------
 from .v2022_01_15_preview.models import *
 from .v2022_04_02_preview.models import *
-from .v2022_11_01.models import *
+from .v2023_05_01.models import *
```

Comment:
>.v2023_05_01. [](http://example.com/codeflow?start=5&length=13)

I am assuming you have not made any changes to the generated files under vendored_sdks folder, and so not reviewing these files.  If you have made any changes, let me know and I will review them